packages/graphql/common.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+require('isomorphic-fetch');
+
+var React = require('react');
+var ReactApollo = require('react-apollo');
+
+var hopsConfig = require('hops-config');
+
+var Context = require('hops-redux').Context;
+
+exports.INTROSPECTION_RESULT = 'INTROSPECTION_RESULT';
+
+exports.Context = exports.createContext = Context.extend({
+  initialize: function (options) {
+    Context.prototype.initialize.call(this, options);
+    this.client = this.createClient(options.graphql || {});
+    this.registerReducer('apollo', this.client.reducer());
+  },
+  createClient: function (options) {
+    return new ReactApollo.ApolloClient(
+      Object.assign(
+        options,
```

Comment:
I Propose to treat `options` as immutable and `Object.assign({}, options, { networkInterface: /*...*/ })`

Reply:
done in 40f67df